centaurCwlRunner/src/main/scala/centaur/cwl/PAPIPreprocessor.scala

```diff
@@ -0,0 +1,121 @@
+package centaur.cwl
+import io.circe.optics.JsonPath
+import io.circe.optics.JsonPath._
+import io.circe.{Json, JsonObject, yaml}
+
+/**
+  * Tools to pre-process the CWL workflows and inputs before feeding them to Cromwell so they can be executed on PAPI.
+  */
+object PAPIPreprocessor {
```

Comment:
ToL: Awesome. I'll refactor this into a generic processor that can feed in a prefix jic we want to test other backends too.